rs/ic_os/vsock/vsock_lib/src/host/agent.rs

```diff
@@ -28,7 +28,7 @@ const NODE_ID_FILE_PATH: &str = "/boot/config/node-id";
 const SETUP_HOSTNAME_FILE_PATH: &str = "/opt/ic/bin/setup-hostname.sh";
 
 // upgrade
-const UPGRADE_FILE_PATH: &str = "/tmp/upgrade.tar.gz";
+const UPGRADE_FILE_PATH: &str = "/tmp/upgrade";
```

Comment:
Why is there now no file extension?

Reply:
Upgrades are agnostic to gz or zst upgrade images, so we are dropping the extension to reflect this. The upgrade works without the file extension 👍